### PR TITLE
moved sass-true to the dev dependencies again

### PR DESCRIPTION
**Pull request by red-freak**
By writing an own error-function it's not necessary any more to load sass-true in production mode.

- used linter to ensure correct linting
- updated sassDoc

### CODE REVIEW

File: helpers/_index.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@forward 'error';
```
Comment (red-freak) [inline helpers/_index.scss:1]:
```suggestion
@forward 'error';

```

File: package.json
Comment (felix-berlin) [inline package.json:1]: If you like you can register here as a contributor.

For this replace the author object with an array like this:
```
contributors: [
{
    "name": "Felix Scholze",
    "email": "felix@webshaped.de",
    "url": "https://webshaped.de"
  },
 {
    "name": "pls add",
    "email": "pls add",
    "url": "pls add"
 }
]
```

**Comment by felix-berlin** [comment]
This works wonderfully! Without the "True" dependency the whole thing feels a bit lighter again ^^

**Review by felix-berlin — CHANGES_REQUESTED** [review]
Hey Sven, fortunately we don't have to create the docs manually. This GH workflow takes the work out of it :)

Can you please remove CNAME and index.html from git?

https://github.com/felix-berlin/sass-butler/blob/master/.github/workflows/build-docs.yml

File: package.json
Comment (red-freak) [reply package.json:1]: thanks! I reverted the sassDoc-commit and added myself as contributor.

**Review by felix-berlin — APPROVED** [review]
👍

**Pull request merged**